docs/source/getting_started/03_custom_nodes.md

```diff
@@ -185,19 +178,35 @@ class CSVLogger:
 
 ### Final Checks
 
-- By default, PeekingDuck assumes that your custom nodes are found in `src/<custom_folder_name>`.
+- [ ] By default, PeekingDuck assumes that your custom nodes are found in `src/<custom_folder_name>`.
+- [ ] NOTE: Create a `results` folder at the same level as `src`!
+- [ ] Ensure that the files are in the correct folder structure.
+
 
-- Ensure that the files are in the correct [folder structure](#folder-structure).
+Your repository should look similar to this:
 
-- NOTE: Create a `results` folder at the same level as `src`!
+```bash
+.
+├── src
+│   └── <custom_folder_name>
+│       ├── output
+│       │    ├── utils
+│       │    │   └── csv.py
+│       │    └── csv_writer.py
+│       └── configs
+│            └── output
+│                └── csv_writer.yml
+├── run_config.yml
+├── results
+```
 
 ### Running the Module
 
 - Now the setup is complete and ready to run! Run the setup using the following code.
 
     `peekingduck run --config_path run_config.yml`
 
-- If the setup is working, you should see an output screen being displayed. Terminate the program by clicking on the output screen and pressing `q`.
+- If the setup is working, you should see an output screen being displayed. If you are using `input.live`, terminate the program by clicking on the output screen and pressing `q`.
```

Comment:
Strongly suggest `pip install .` with the latest `dev` and run through all these steps (with the updated changes) to make sure it works.

Reply:
should this be part of the test later?

Reply:
Okay